app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1> <%= @guest.name %>'s Page </h1>
+<br>
+Rooms:
+<br>
+<% @guest.rooms.each do |room| %>
+  <li> Rate: $<%= room.rate %>.00, Suite: <%= room.suite %>, Hotel: <%= room.hotel.name %></li> 
+  <br>
+<% end %>
+
+<%= form_with url: "/guests/#{@guest.id}/guest_rooms", method: :post, local: true do |f| %>
+  <%= f.label "Suite ID: "  %>
+  <%= f.number_field :room_id %>
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
I will touch on this in the `routes.rb` file, but the URL for the form isn't correct according to REST.

```ruby
<%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |f| %>
```

Also, your label needs to also have a `name` attribute that is the same as the field that the label is for.

```ruby
<%= f.label :room_id,  "Suite ID: " %>
```